services/elasticache/pkg/resource/cache_parameter_group/custom_api.go

```diff
@@ -92,6 +92,7 @@ func (rm *resourceManager) provideEvents(
 	resp, err := rm.sdkapi.DescribeEventsWithContext(ctx, input)
 	rm.metrics.RecordAPICall("READ_MANY", "DescribeEvents-CacheParameterGroup", err)
 	if err != nil {
+		rm.log.Error(err, "Error during DescribeEvents-CacheParameterGroup")
```

Comment:
I generally dislike using the log.Error call anywhere other than the root calling location. The controller-runtime will automatically log any error it gets back from the resourceManagers, so why not just return nil, respErr here and remove the log.Error calls

Reply:
The error messages logged here were helpful in debugging.
Please revisit.

Reply:
Please change to `rm.log.V(1)` then. The error is already logged higher in the reconciler.

Reply:
Used `rm.log.V(1).Info`